### PR TITLE
Drop foreman-sqlite

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

**Comment by tbrisker** [comment]
I think failure is due to core still requiring mysql

File: debian/bionic/foreman/control [hunk 1]
```diff
@@ -20,7 +20,7 @@ Depends: ruby, ruby2.5, bundler (>= 1.3), zlib1g-dev,
 Recommends: foreman-proxy, foreman-debug (= ${binary:Version}), python-minimal
 Conflicts: ruby-activerecord-deprecated-finders, ruby-foreman-tasks (<< 0.10.1), ruby-sequel-pg
 Replaces: foreman-compute
-Breaks: foreman-compute, foreman-mysql2
+Breaks: foreman-compute, foreman-mysql2, foreman-sqlite
```
Comment (ekohl) [inline debian/bionic/foreman/control:23]:
```suggestion
Breaks: foreman-compute, foreman-mysql2, foreman-sqlite3
```

**Comment by mmoll** [comment]
Nope. This should turn green, once a tarball without sqlite requirements is used.

File: debian/bionic/foreman/control (hunk 1 above)
Comment (tbrisker) [reply debian/bionic/foreman/control]: good catch, fixed

**Comment by tbrisker** [comment]
> Nope. This should turn green, once a tarball without sqlite requirements is used.

that's what i meant by core still requiring sqlite - the core PR hasn't been merged yet.

**Comment by mmoll** [comment]
[test packaging]

**Comment by mmoll** [comment]
merged, thanks @tbrisker!

**Pull request merged**